textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTester.java

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
```

Comment:
I personally favor to keep it as a separate class. And I think RegexMatcherTestHelper is a better name.